src/pages/Documents/DocumentsTable.tsx

```diff
@@ -175,23 +178,33 @@ const DocumentsTable = () => {
               </Button>
             ) : (<> </>)
             }
+            {(canEdit == true || regionAdm == true || cityAdm == true || clubAdm == true || supporter == true || plastMember == true) ? (
              <Search
                 enterButton
                 placeholder="Пошук"
                 allowClear
                 onChange={handleSearchChange}
                 onSearch={handleSearch}                
                />
-             </div>
+            ) : (<> </>)
+            }
+            </div>
 
-             {loading ? (<Spinner />) :<Card
+          {(canEdit == true || regionAdm == true || cityAdm == true || clubAdm == true || supporter == true || plastMember == true) ? (
+          <Card
             style={{ width: '100%' }}
             tabList={tabList}
             activeTabKey={status}
             onTabChange={(key) => {
             onTabChange(key);
-            }}
-          >
+            }} 
+          /> ) : (
+          <Card
+              style={{ width: '100%' }}
```

Comment:
Are you sure that it's an adaptive Cart? 
Because you use  width: 100%, it's bad practise

Reply:
Yes, we can scroll it as in other pages